classes/Player.py

```diff
@@ -4,4 +4,9 @@ class Player(pygame.sprite.Sprite):
     def __init__(self,groups):
         super().__init__(groups)
         self.image=pygame.image.load('./graphics/player.png').convert_alpha()
-        self.rect=self.image.get_rect(center=(window_width/2,window_height/2))
\ No newline at end of file
+        self.rect=self.image.get_rect(center=(window_width/2,window_height/2))
+    def input_position(self):
+        pos=pygame.mouse.get_pos()
+        self.rect.center=pos
+    def update(self):
```

Comment:
update is calling input position method with 2 lines unit.
I would suggest removing this input position method and refactor the update method.